packages/vscode-dependencies-validation/src/editorChanges.ts

```diff
@@ -58,10 +64,15 @@ function executeRefreshDiagnosticsOnDocumentChangeEvent(
   diagnosticCollection: DiagnosticCollection
 ) {
   return (event: TextDocumentChangeEvent) => {
-    // this event gets called multiple (four) times for each text change
-    // but only once for "true" text changes
-    if (!isEmpty(event.contentChanges)) {
-      void refreshDiagnostics(event.document.uri, diagnosticCollection);
+    if (
+      // this event gets called multiple (four) times for each text change
+      // but only once for "true" text changes
+      !isEmpty(event.contentChanges) &&
+      shouldBeChecked(event.document.uri.path)
+    ) {
+      const uri = event.document.uri;
```

Comment:
same as above

Reply:
here I always have the document so I corrected as you suggested.